icu4j/main/common_tests/src/test/java/com/ibm/icu/dev/test/format/MeasureUnitGeneratorTest.java

```diff
@@ -0,0 +1,845 @@
+// © 2016 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+/*
+ *******************************************************************************
+ * Copyright (C) 2013-2016, International Business Machines Corporation and
+ * others. All Rights Reserved.
+ *******************************************************************************
+ */
+package com.ibm.icu.dev.test.format;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.TreeMap;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import com.ibm.icu.dev.test.CoreTestFmwk;
+import com.ibm.icu.impl.Pair;
+import com.ibm.icu.util.MeasureUnit;
+import com.ibm.icu.util.NoUnit;
+
+/**
+ * This is not a real test class. It is only used to
+ * generate updated unit tests code based on new CLDR data.
+ * Do not add any other tests here.
+ *
+ * See https://unicode-org.github.io/icu/processes/unicode-update.html
```

Comment:
```suggestion
 * See https://unicode-org.github.io/icu/processes/release/tasks/standards.html
```

Reply:
There is nothing relevant at https://unicode-org.github.io/icu/processes/release/tasks/standards.html

Only a title ("Update MeasureUnit Code") with a link (https://github.com/unicode-org/icu/blob/main/docs/processes/release/tasks/updating-measure-unit.md)

Which is the link I use here, bypassing one level of indirection.

Reply:
But you are pointing to https://unicode-org.github.io/icu/processes/unicode-update.html which is unrelated. Therefore I looked at the replacement page for the original Google Sites URL.

Reply:
Oh, right!

What do you think about using 
https://github.com/unicode-org/icu/blob/main/docs/processes/release/tasks/updating-measure-unit.md
?

Reply:
ok

Reply:
DONE